### ISSUE TITLE
USER->VIEW CART  No label of the total price of the selected cart items 

### ISSUE THREAD

**Issue by algreg1998** [ploidddddd/purplekeep#116]
![image](https://user-images.githubusercontent.com/20896859/36766359-769bf162-1c70-11e8-9b77-e8df80ed18d2.png)

becarefull beacuse this page is full ajax. you might destroy it

**Comment by marykrisv** [comment]
952a92823dbdcc024f909fefcdb1887472627a8d

**Comment by marykrisv** [comment]
commit f585c25112cfbff00c755d0104d929ec3dacdffd 

nag change kog branch sir

ni tabang pd si Algreg ani sir
